pkg/net/libp2p/key_test.go

```diff
@@ -136,14 +136,17 @@ func TestNetworkPublicKeyToOperatorPublicKey(t *testing.T) {
 		t.Errorf("operator public key uses the wrong curve")
 	}
 
-	if operatorPublicKey.X.Cmp(
-		networkPublicKey.(*libp2pcrypto.Secp256k1PublicKey).X,
-	) != 0 {
+	publicKey, ok := networkPublicKey.(*libp2pcrypto.Secp256k1PublicKey)
```

Comment:
s/publicKey/libp2pPublicKey

Reply:
Done in ddf70b772887607d03e76dcecdbb7a1c6f55ae1b.